templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ -n "${1}" ]
+then
+	LIFERAY_DOCKER_UBUNTU_PRO_TOKEN="${1}"
+fi
+
+if [ -z "${LIFERAY_DOCKER_UBUNTU_PRO_TOKEN}" ]
+then
+	echo "Ubuntu Pro subscription attachment is skipped as LIFERAY_DOCKER_UBUNTU_PRO_TOKEN is not set."
+
+	exit 1
+fi
+
+if (pro status 2>/dev/null | grep -q "Subscription: Ubuntu Pro" 2>/dev/null)
+then
+	echo "Ubuntu Pro subscription is already active."
+
+	exit 0
+fi
+
+apt-get update && \
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes ubuntu-advantage-tools && \
+pro attach "${LIFERAY_DOCKER_UBUNTU_PRO_TOKEN}"
```

Comment:
@zsoltbalogh, shouldn't we use the same pattern BChan uses to build the scripts? I mean, have a function named `main` which is the entrypoint of the script? For example: https://github.com/liferay/liferay-docker/blob/d735f8d188b71c72be90cfc57a5951179417eadb/templates/bundle/resources/usr/local/bin/configure_liferay.sh#L104